readthedocs/config/config.py

```diff
@@ -146,6 +146,8 @@ class BuildConfigBase:
         'mkdocs',
         'submodules',
     ]
+    valid_build_images = [k.split(':')[1] for k in DOCKER_IMAGE_SETTINGS]
```

Comment:
Each version has different valid images, v1 accepts number versions, v2 only latest and stable (from the user side).

Reply:
Means we can still override them from the admin

Reply:
Mmm... this is interesting. I wasn't aware.

Reply:
> v2 only latest and stable (from the user side).

Isn't this line saying that `3.0` and other numbered versions are supported also: https://github.com/rtfd/readthedocs.org/blob/master/readthedocs/config/config.py#L580

Reply:
Yeah, but they aren't documented. I'm not sure if we will allow users to specify that from the config file, it wasn't clear to me.

Reply:
From what I linked in the comment above, I guess no